### PR TITLE
new icon: firebird (original, plain)

### PR DESCRIPTION
**Pull request by Bulani**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://www.firebirdsql.org/en/logos/

### CODE REVIEW

File: devicon.json
Comment (canaleal) [inline devicon.json:1]: Great work on the JSON everything looks good.

File: icons/firebird/firebird-original.svg
Comment (canaleal) [inline icons/firebird/firebird-original.svg:1]: The SVG is not scaled to 128x128
It is also optimized and does not reach the edges of the viewbox. 

Please refer to the wiki to read more about the SVG requirements. 
https://github.com/devicons/devicon/wiki/SVG-Standards

File: icons/firebird/firebird-plain.svg
Comment (canaleal) [inline icons/firebird/firebird-plain.svg:1]: This SVG also has to be scaled and optimized.

**Review by canaleal — COMMENTED** [review]
Thankyou for your contribution! Ive added some comments on what needs to be changed. If you need any help feel free to ask.

**Comment by Bulani** [comment]
@canaleal I resolveded the problems in icons

**Review by canaleal — APPROVED** [review]
I scaled up the SVGs to have a viewbox of 128x128. 

There are no artifacts in the SVGs. 

image

**Pull request merged**